aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md

```diff
@@ -257,7 +257,7 @@ Open the `CreateUserWizardWithRoles.aspx` page and add a CreateUserWizard contro
 
 Next, select the "Add/Remove `WizardSteps`…" option from the CreateUserWizard's Smart Tag and add a new `WizardStep`, setting its `ID` to `SpecifyRolesStep`. Move the `SpecifyRolesStep WizardStep` so that it comes after the "Sign Up for Your New Account" step, but before the "Complete" step. Set the `WizardStep`'s `Title` property to "Specify Roles", its `StepType` property to `Step`, and its `AllowReturn` property to False.
 
-[![Add the](assigning-roles-to-users-cs/_static/image32.png)](assigning-roles-to-users-cs/_static/image31.png)
+[![Add a new Wizard Step.](assigning-roles-to-users-cs/_static/image32.png)](assigning-roles-to-users-cs/_static/image31.png)
```

Comment:
Let's make it a little more descriptive. Something like "Screenshot that shows the selected Specify Roles properties in the Wizard Step Collection Editor window."

Reply:
Addressed.